### ISSUE TITLE
[BUG]: Cannot find package 'lru-cache' imported from /var/task/node_modules/@octokit/auth-app/dist-node/index.js

### ISSUE THREAD

**Issue by gaomingzhao666** [octokit/octokit.js#2756]
### What happened?

Hello, I am using `Octokit` to build my portfolio to show my repos from `Github`, and today I accessed the website that deployed on `Vercel` and I got the error info as the following: 

```sh
[nuxt] [request error] [unhandled] [500] Cannot find package 'lru-cache' imported from /var/task/node_modules/@octokit/auth-app/dist-node/index.js
```

Here is my repo's [link](https://github.com/gaomingzhao666/nano-portfolio), I wrote all of the codes that depends on `Octokit` in the `Server -> api -> repo` folder 

The error is shown in the `Vercel log` and it works properly locally, I have no idea about it, I tried to redeploy on `Vercel` but it doesn't work.

My Octokit version is 4.0.2. It works properly with this version in local. I would very much appreciate any help!!

### Versions

Octokit version 4.0.2
NodeJS version 20.X LTS

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

**Comment by wolfy1339** [comment]
This seems like a bug in vercel. We're using npm specifiers to use a fork of one of our dependencies,
https://github.com/octokit/auth-app.js/blob/70e2904e2f1d5004e12c685ef86276f0e3a0936d/package.json#L34

make sure that vercel resolves that package to the correct one

**Comment by gaomingzhao666** [comment]
> This seems like a bug in vercel. We're using npm specifiers to use a fork of one of our dependencies, https://github.com/octokit/auth-app.js/blob/70e2904e2f1d5004e12c685ef86276f0e3a0936d/package.json#L34
> 
> make sure that vercel resolves that package to the correct one

Thank you so much sir, I have reported it to `Vercel Community`. (●'◡'●)

**Comment by gaomingzhao666** [comment]
The issue was solved by installing @octokit/core instead of octokit.js, I do not know why it happened and where the problem should be addressed.

I found the [issue](https://github.com/nuxt/nuxt/issues/22325#issuecomment-1690421926) but regenerate lockfile or redeploy project can not handle this.

The codes that explicitly point out the lost file path as following:

```ts
  // relate to issue https://github.com/nuxt/nuxt/issues/22325#issuecomment-1690421926
  nitro: {
    externals: {
      traceInclude: []
    }
  },
```

However I did not find the lost folder called `auth-app` in node_modules.
